src/js/components/ConfirmDialog/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react';
+import {Modal, ModalHeader, ModalBody, ModalFooter, Button} from 'reactstrap';
+import PropTypes from 'prop-types';
+
+class ConfirmDialog extends Component {
+
+  onContinue = () => {
+    this.props.onConfirm();
+    this.props.onCancel();
```

Comment:
Shouldn't this be only confirm?

Reply:
I used the onCancel() there to close the dialog right after calling the onConfirm.

Reply:
Ah ok. You shouldn't do that there, as the dialog itself doesn't know what `onConfirm` and `onCancel` do. 

Example, I could create a component that uses the dialog and on confirm, send a TX, but on cancel, switch page. Now that would be confusing because it would do both.

What you should do is juste handle the close in the other component (you can in that other component do both confirm and cancel if you like because it's the component itself that handles it and it knows that cancel only closes)